src/Declaration.js

```diff
@@ -110,9 +110,15 @@ export class SyntheticDefaultDeclaration {
 			return this.original.run( strongDependencies );
 		}
 
-		if ( /FunctionExpression/.test( this.node.declaration.type ) ) {
-			return run( this.node.declaration.body, this.statement.scope, this.statement, strongDependencies, false );
+		let declaration = this.node.declaration;
+		while ( declaration.type === 'ParenthesizedExpression' ) declaration = declaration.expression;
+
+		if ( /FunctionExpression/.test( declaration.type ) ) {
+			return run( declaration.body, this.statement.scope, this.statement, strongDependencies, false );
 		}
+
+		// otherwise assume the worst
+		return true;
```

Comment:
yeah – previously we were assuming that if you called the `foo` in `import foo from './foo.js'`, the default export from `foo.js` was either a reference to another declaration, or a FunctionExpression or ArrowFunctionExpression. But that might not be true:

```js
export default Math.random()  < 0.5 ? x => x * 2 : x => x * 3;
```

We're not *currently* equipped to handle complex cases like that, so we should err on the side of caution. The ParenthesizedExpression stuff (which this was originally designed to address) is really just an optimisation.